### PR TITLE
Improve setting-up.md documentation #8920 

### LINKED ISSUE #8920: improve setting-up.md documentation
The initial development environment setup documentations has some space for improvement. It can be written in shorter and clearer sentences.

### PR DESCRIPTION
**Pull request by cybersol795**
Fixes #8920 
Typos and grammatical errors are removed and some sentence structures have slightly changed in some sections. Please review the revisions and if there is any change that you don't approve please leave me a comment. Thank you!

p.s., if you approve, after this issue is closed, I will create ~3 more issues for a few other documentation files in the repository (but will create and close them one by one)

### CODE REVIEW

File: docs/setting-up.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Setting up a development environment
 
-This is a step-by-step guide to setting up a development environment in your local machine.
-You will use that environment to work on features, enhancements, bug fixes, etc. which ultimately allows you to contribute to the project.
+This is a step-by-step guide for setting up a development environment in your local machine.
```
Comment (tshradheya) [inline docs/setting-up.md]: `on` your local machine?

File: docs/setting-up.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
...
 1. Install Node.js (minimum version 6.11.5).
 
-## Step 2: Obtain your own repository copy
+## Step 2: Obtain your own copy of the repository
 
-1. Fork our repo at https://github.com/TEAMMATES/teammates. Clone that fork to your hard disk.
+1. Fork our repo at https://github.com/TEAMMATES/teammates. Clone the fork to your hard disk.
 
-1. Add a remote name (e.g `upstream`) for the main repo for your repo to keep in sync with, and then fetch the remote-tracking branches from the main repo.
+1. Add a remote name (e.g upstream) for your copy of the main repo. Fetch the remote-tracking branches from the main repo to keep it in sync with your copy.
```
Comment (tshradheya) [inline docs/setting-up.md]: Let's keep the ***`*** around upstream as its a special word in this context

File: docs/setting-up.md (hunk 1 above)
Comment (cybersol795) [reply docs/setting-up.md]: yes, "on your local machine" makes more sense.

File: docs/setting-up.md (hunk 2 above)
Comment (cybersol795) [reply docs/setting-up.md]: I missed the single quotes. Will change it.

File: docs/setting-up.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ More information can be found at [this documentation](https://help.github.com/ar
 1. Install Google Cloud SDK. Follow the directions given [here](https://cloud.google.com/sdk/downloads).
    Note that you *do not* need to [initialize the SDK](https://cloud.google.com/sdk/docs/initializing).
    ```sh
-   # This command is to be run at the Google Cloud SDK directory
+   # Execute this command at the Google Cloud SDK directory
```
Comment (tshradheya) [inline docs/setting-up.md]: Lets keep this consistent with line 61.

Change to ` Run the following command`

**Comment by tshradheya** [comment]
Thanks for the contribution @cybersol795 . Looking forward to the future PR's which aim to improve the documentation.

Also in future please create a new branch and then create a PR from that :)

**Pull request merged**